apiary.apib

```diff
@@ -3400,6 +3404,9 @@ Update a draft invoice. Booked invoices cannot be updated.
         + expected_payment_method (ExpectedPaymentMethod, optional, nullable)
         + custom_fields (array[CustomFieldValue], optional)
         + document_template_id: `179e1564-493b-4305-8c54-a34fc80920fc` (string, optional)
+        + invoice_currency (object, optional)
```

Comment:
Maybe we should expand on what the default would be if not given?

```suggestion
        + invoice_currency (object, optional) - Defaults to department currency when not given
```

Reply:
I definitely need more context give an opinion. Is there a JIRA ticket to read?

Reply:
Yes, sure: https://teamleader.atlassian.net/browse/ATL-2189 and https://teamleader.atlassian.net/browse/ATL-2193

Reply:
exchange rate changes daily? Why would someone store it in teamleader